### PR TITLE
Move the nametag back to the top of the player

### PR DESCRIPTION
**Pull request by TeTpaAka**
See https://github.com/minetest/minetest/issues/6174

### CODE REVIEW

**Comment by nerzhul** [comment]
it's not the correct fix, as now we can change model, it's not a fixed size like before. Can you find the correct value of the model instead ?

**Comment by TeTpaAka** [comment]
Updated.

File: src/camera.h [hunk 1]
```diff
@@ -157,7 +160,8 @@ class Camera
 	}
 
 	Nametag *addNametag(scene::ISceneNode *parent_node,
-		std::string nametag_text, video::SColor nametag_color);
+		std::string nametag_text, video::SColor nametag_color,
```
Comment (nerzhul) [inline src/camera.h]: nice, can you just fix to use const ref on the string and the v3f please ?

Comment (TeTpaAka) [inline src/camera.h]: fixed

File: src/content_cao.cpp [hunk 2]
```diff
@@ -895,8 +895,11 @@ void GenericCAO::addToScene(ITextureSource *tsrc)
 	scene::ISceneNode *node = getSceneNode();
 	if (node && m_prop.nametag != "" && !m_is_local_player) {
 		// Add nametag
+		v3f pos;
+		pos.Y = m_prop.collisionbox.MaxEdge.Y + 0.1f;
```
Comment (paramat) [inline src/content_cao.cpp]: Probably needs to be higher, as before it was position + 1.1, which is 2.1 above foot position, the collisionbox top is at 1.77 above foot position, so add 0.3f?

Comment (TeTpaAka) [inline src/content_cao.cpp]: fixed

**Pull request merged**